crt_portal/static/js/save_warning.js

```diff
@@ -0,0 +1,16 @@
+(function() {
+  let showWarning = false;
+  const forms = document.querySelectorAll('.usa-form');
+  forms.forEach(form => {
+    form.addEventListener('change', () => (showWarning = true));
+  });
+  const btns = document.querySelectorAll('.usa-button');
```

Comment:
usa-button might be overselecting, here.

![uhoh](https://github.com/usdoj-crt/crt-portal/assets/15126660/e3911349-226f-4a4b-b0b2-ec83b5fdf24f)

maybe something like "type=submit" would be a better selection?